CondFormats/DataRecord/interface/HGCalMappingCellIndexerRcd.h

```diff
@@ -0,0 +1,26 @@
+#ifndef CondFormatsCellIndexerRcd_HGCalMappingCellIndexerRcd_h
+#define CondFormatsCellIndexerRcd_HGCalMappingCellIndexerRcd_h
+// -*- C++ -*-
+//
+// Package:     CondFormats/DataRecords
+// Class  :     HGCalMappingCellIndexerRcd
+//
+/**\class HGCalMappingCellIndexerRcd HGCalMappingCellIndexerRcd.h CondFormats/DataRecords/interface/HGCalMappingCellIndexerRcd.h
+
+ Description: [one line class summary]
+
+ Usage:
+    <usage>
+
+*/
```

Comment:
Since we're here... can you please fill the description and usage, or else remove these comments ?